preview/feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -85,7 +85,6 @@
                 <extension>org.wildfly.extension.io</extension>
                 <extension>org.wildfly.extension.messaging-activemq</extension>
                 <extension>org.wildfly.extension.metrics</extension>
-                <extension>org.wildfly.extension.micrometer</extension>
```

Comment:
Please add this back.

Reply:
Brian ended up addressing this in a PR against this PR. :)

Reply:
Brian ended up addressing this in a PR against this PR. :)